### DISCUSSION TITLE
Change generator meta tag?

### DISCUSSION THREAD

**Discussion by aukc1970** [getformwork/formwork#937] [Q&A]
I thought that it is possible to change or not to show this tag by means of functions but it is not clear they work or not ? metadata.set_generator true to show this meta tag and false to not show this meta tag but it doesn't work. Or, you can override generator: Workflow # Only if ' metadata.set_generator` is true so it is written in wiki. Here are links to how it's done in GravCMS.
https://discourse.getgrav.org/t/change-generator-meta-tag/2772
https://discourse.getgrav.org/t/generator-tag/1916

**Comment by giuscris** [comment]
Oh that's possible, you can override the generator in your `site.yml` configuration file. E.g.:

```yaml
metadata:
    generator: Whatever you want
```

Then it renders to:

```html
<meta name="generator" content="Whatever you want">
```
